main.py

```diff
@@ -28,9 +27,13 @@ def if_statements():
 
 
 @app.command()
-def looping_constructs():
+def looping_constructs(directory_path):
     """Determine number of looping constructs in a Python directory."""
-    print("In progress")
+    cast_dict = generate_trees(directory_path)
+    while_loops_dict = counter.count_whileloops(cast_dict)
+    for_loops_dict = counter.count_forloops(cast_dict)
+
+    print(while_loops_dict, for_loops_dict)
```

Comment:
Once the `total_loops` function is completed in `counter.py`, this print statement should be changed to output the total number of looping constructs in a directory.